src/apps/backend/modules/comments/types.ts

```diff
@@ -0,0 +1,64 @@
+import { Account } from '../account';
+import { ApplicationError } from '../application';
+import { HttpStatusCodes } from '../http';
+// import { Task } from '../task';
+
+
+export class Comment {
+    id: string;
+    task: string;
+    account: string | Account;
+    comment: String;
+    createdAt: Date;
+    updatedAt: Date;
+  
+    constructor(
+      id: string,
+      task: string,
+      account: string | Account,
+      comment: String,
+      createdAt: Date,
+      updatedAt: Date,
+    ) {
+      this.id = id;
+      this.task = task;
+      this.account = account;
+      this.comment = comment;
+      this.createdAt = createdAt;
+      this.updatedAt = updatedAt;
+    }
+  }
+
+export interface CreateCommentParams {
+    taskId: String;
+    accountId:String;
+    comment: String;
```

Comment:
Always use `string` instead of `String` when defining string types in TypeScript.